src/main/java/org/elasticsearch/gateway/GatewayMetaState.java

```diff
@@ -530,7 +533,7 @@ public void run() {
                 }
                 logger.warn("[{}] deleting dangling index", metaData.index());
                 try {
-                    indicesService.deleteIndexStore("deleting dangling index", metaData);
+                    indicesService.deleteIndexStore("deleting dangling index", metaData, clusterService.state());
```

Comment:
I got to this change when I tried to remove all the options around dangling indices, this code will go away soon :)